src/Select.soy

```diff
@@ -18,18 +19,21 @@
 	<div class="select{$elementClasses ? ' ' + $elementClasses : ''}" data-onkeydown="handleKeyDown_">
 		{let $currSelectedIndex: isNonnull($selectedIndex) ? $selectedIndex : ($label or (length($items) == 0) ? -1 : 0) /}
 		<input
+			disabled="{$disabled}"
 			type="hidden"
 			name="{$hiddenInputName ? $hiddenInputName : ''}"
 			value="{$currSelectedIndex == -1 ? '' : $values ? $values[$currSelectedIndex] : ''}"
 		>
 		{call Dropdown.render}
 			{param body kind="html"}
 				{foreach $item in $items}
-					<li
-						data-onclick="{$handleItemClick_ ?: ''}"
-						data-onkeydown="{$handleItemKeyDown_ ?: ''}"
-						class="select-option{$currSelectedIndex == index($item) ? ' selected' : ''}">
-						<a href="javascript:;">{$item}</a>
+					<li class="select-option{$currSelectedIndex == index($item) ? ' selected' : ''}">
+						<button
```

Comment:
We're following bootstrap's markup and css, so it's best to keep using `a` as before so that we keep being the same. What bootstrap does for disabled dropdown items is to add a `disabled` class to the `li` tag. That will give the expected bootstrap style without us having to try to write it again for `button`. Could you do that instead of changing to `button`?